orderbook/src/database/orders.rs

```diff
@@ -56,6 +57,30 @@ impl DbOrderKind {
     }
 }
 
+/// Location for which the sellAmount should be drawn upon order fulfilment
+#[derive(sqlx::Type)]
+#[sqlx(type_name = "BalanceFrom")]
+#[sqlx(rename_all = "snake_case")]
+pub enum BalanceFrom {
+    /// Order owner's ethereum address
+    Owner,
```

Comment:
A more accurate description is that the balance is taken using direct ERC20 allowances to the vault relayer contract. I find the names/description confusing as the balance always comes from the order's owner, it just comes from difference sources:
- Direct ERC20 allowances to the Vault relayer contract
- ERC20 allowances to the Vault with GPv2 relayer approval
- Internal balances to the Vault with GPv2 relayer approval

The reason I feel like "Order owner's ethereum address" is ambiguous is that it makes me think that the address holds the balance, which isn't entirely accurate, as its the ERC20 token contract that keeps track of the balances in storage for both `BalanceFrom::Owner` and `BalanceFrom::VaultExternal` (as opposed to ETH balance which is tied to the account and thus "held by the owner's ethereum address") or the Vault contract for `BalanceFrom::Internal`.